### DISCUSSION TITLE
Easier integration with external issue trackers

### DISCUSSION THREAD

**Discussion by Martmists-GH** [community/community#20523] [Projects and Issues]
Currently if you want to use a custom issue tracker (YouTrack, JIRA, etc.) you have two options to inform your users:

- Disable the issues tab and add a line in the README linking to your custom issue tracker
- Keep issues enabled, but add a template that links to your issue tracker

A better solution for this use case would be to allow configuring an URL in the project settings to be an external issue tracker, which would show the Issues tab with the "external url" icon similar to the second option mentioned before.

It would look something like this:
![image](https://user-images.githubusercontent.com/16361449/178997178-d0829231-db0b-40ba-9f23-2fa20727e707.png)

**Comment by pboushy** [comment]
GitLab has this capability and it'd be great to have this in GitHub.
